fedimint-core/src/util/backoff_util.rs

```diff
@@ -11,22 +11,12 @@ pub fn background_backoff() -> FibonacciBackoff {
 }
 
 /// A backoff strategy for relatively quick foreground operations.
-///
-/// Starts at 200ms and increases to 5s. Will retry 10 times before giving up,
-/// with a maximum total delay between 20.8s and 22.8s depending on jitter.
```

Comment:
20s is too low, and that math was cool, but I'm not doing it again. :D

Reply:
That's kind of a problem with fibonnaci retries. Not that easy to ballpark the total time after N retries.